app/Models/Mship/Account.php

```diff
@@ -512,4 +514,35 @@ public function __toString()
     {
         return $this->name;
     }
+
+    /**
+     * Return a valid access token for the user or return null if none.
+     *
+     * @return \League\OAuth2\Client\Token\AccessToken
+     * @return null
```

Comment:
Does it always return null?

Reply:
Line 521 gives you the other possible return type

Reply:
senior moment...